vignettes/multiple-biomes.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+title: Running Hector with multiple biomes
+author: Alexey Shiklomanov
+date: "2019-06-27"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Running Hector with multiple biomes}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+By default, Hector's carbon cycle model treats the entire land surface as a single, homogeneous ecosystem.
+However, it is possible to introduce some land surface heterogeneity by splitting the land surface into several different biomes with distinct parameters.
+In this vignette, we show two examples of performing multi-biome analyses with Hector.
+
+This vignette assumes some familiarity with the Hector model and its R interface.
+For help on the former, consult the [Hector manual](manual/index.html), and on the latter, see the [introductory vignette](intro-to-hector.html).
+
+Before we begin, let's load the R package we will need: the `hector` package to interact with Hector, and the `ggplot2` package for plotting.
+
+```{r}
+library(hector)
+library(ggplot2)
+```
+
+# Example 1: Accelerated high-latitude warming
+
+Observational and modeling studies have found that high-latitudes, particularly in the northern hemisphere, are warming significantly faster than the global average.
```

Comment:
"high-latitude ecosystems" or "high latitudes"

Reply:
Addressed in 551e47f366caff261080a4476d3d556960a14ee8.

Reply:
Addressed in 551e47f366caff261080a4476d3d556960a14ee8.